linking problem on OSX

we're [getting](https://travis-ci.org/CCPPETMR/SIRF/jobs/465480695)
```
Undefined symbols for architecture x86_64:
  "Gadgetron::NFFT_plan<Gadgetron::hoNDArray, float, 2u>::compute_NFFT_C2NC(Gadgetron::hoNDArray<Gadgetron::complext<float> >&, Gadgetron::hoNDArray<Gadgetron::complext<float> >&)", referenced from:
      vtable for Gadgetron::hoNFFT_plan<float, 2u> in hoNFFT_test.cpp.o
```
Probably v3.17.0 was ok.

I have no idea how this can happen.

--- a/toolboxes/nfft/NFFT.h
+++ b/toolboxes/nfft/NFFT.h
@@ -151,14 +151,14 @@ namespace Gadgetron {
     protected:
 
         // Dedicated computes
-        virtual void compute_NFFT_C2NC(ARRAY <complext<REAL>>&in, ARRAY <complext<REAL>>& out);
+        void compute_NFFT_C2NC(ARRAY <complext<REAL>>&in, ARRAY <complext<REAL>>& out);
 
-        virtual void compute_NFFT_NC2C(const ARRAY <complext<REAL>>& in, ARRAY <complext<REAL>>& out);
+        void compute_NFFT_NC2C(const ARRAY <complext<REAL>>& in, ARRAY <complext<REAL>>& out);
 
 
-        virtual void compute_NFFTH_NC2C(const ARRAY <complext<REAL>>& in, ARRAY <complext<REAL>>& out);
+        void compute_NFFTH_NC2C(const ARRAY <complext<REAL>>& in, ARRAY <complext<REAL>>& out);
 
-        virtual void compute_NFFTH_C2NC(ARRAY <complext<REAL>>& in, ARRAY <complext<REAL>>& out);
+        void compute_NFFTH_C2NC(ARRAY <complext<REAL>>& in, ARRAY <complext<REAL>>& out);
 
 
         typename uint64d<D>::Type matrix_size;          // Matrix size

